articles/storage/blobs/storage-quickstart-blobs-go.md

```diff
@@ -214,7 +214,7 @@ for pager.More() {
 
 ### Download the blob
 
-The code sample downloads a blob using the [DownloadStream](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azblob#Client.DownloadStream) method, and creates a retry reader for reading data. If a connection fails while reading, the retry reader makes other requests to re-establish a connection and continue reading. You can specify retry reader options using the [RetryReaderOptions](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/blob#RetryReaderOptions) struct.
+The code sample downloads a blob using the [DownloadStream](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azblob#Client.DownloadStream) method. If a connection fails, the client will re-try the download according to the [RetryOptions](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore/policy#RetryOptions) specified when creating the client.
```

Comment:
The retry policy will retry IFF the response to the initial request fails.  If that succeeds, then the HTTP response is returned.  However, if during reading of the HTTP response body, an error is encountered, the retry policy _won't_ have any effect.  This is the scenario that the retry reader helps with, the download being interrupted mid-stream.

Reply:
Ah ok. I get it. The connection attempt is retried and the download itself, too.
Should this be elaborated more here? I would then revert my changes and add some additional information.